minkindr_conversions/include/minkindr_conversions/kindr_msg.h

```diff
@@ -17,6 +18,14 @@ void poseKindrToMsg(const kindr::minimal::QuatTransformation& kindr,
                     geometry_msgs::Pose* msg);
 void poseMsgToKindr(const geometry_msgs::Pose& msg,
                     kindr::minimal::QuatTransformation* kindr);
+void poseKindrToMsg(const kindr::minimal::QuatTransformation& kindr,
```

Comment:
No, that doesn't follow the convention of the rest of the messages, which
follows the eigen and tf conversions convention in ROS
On Dec 9, 2015 7:23 PM, "Titus Cieslewski" <notifications@github.com> wrote:

> In minkindr_conversions/include/minkindr_conversions/kindr_msg.h
> <https://github.com/ethz-asl/minkindr_ros/pull/4#discussion_r47129767>:
>
> > @@ -17,6 +18,14 @@ void poseKindrToMsg(const kindr::minimal::QuatTransformation& kindr,
> >                      geometry_msgs::Pose* msg);
> >  void poseMsgToKindr(const geometry_msgs::Pose& msg,
> >                      kindr::minimal::QuatTransformation* kindr);
> > +void poseKindrToMsg(const kindr::minimal::QuatTransformation& kindr,
>
> this is not to, but from kindr. So, kindrToPoseStampedMsg, ok?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethz-asl/minkindr_ros/pull/4/files#r47129767>.
>